pages/common/git-authors.md

```diff
@@ -0,0 +1,17 @@
+# git authors
+
+> Display log or list of committers of a Git repository. If you aren't using the list command, a new file called AUTHORS will be created. Sorted by amount of commits.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-authors>.
+
+- Display log of committers of the current Git repository, reading from the AUTHORS file, opened in your default editor:
+
+`git authors`
+
+- Print full list of committers of the current Git repository:
+
+`git authors --list`
+
+- Display full log of committers of the current Git repository, excluding emails, reading from the AUTHORS file, opened in your default editor:
```

Comment:
```suggestion
- Display a log of committers, exluding emails, in the default editor, reading from the `AUTHORS` file:
```

Reply:
why does this have the default editor bit but the other suggestion doesn't?
Consistency isn't present here